### ISSUE TITLE
Question about using wonderwords to generate random prompt info

### ISSUE THREAD

**Issue by guming3d** [Azure/azure-openai-benchmark#9]
This is a great tool, I read the code and found the tool is using wonderwords libs to generate random words and send it to backend model, my question is "Is it make sense to send no logic sentences or words to GPT model to get the response benchmark?" The response time may quite different between prompt with logic to just random words. Not sure the tool is taking the right method. Any comments on this?

**Comment by technicianted** [comment]
Yes you are right. Using random words does not produce equivalent work on the engine backend. However, the purpose of the tool is to help with volume benchmarking rather than accurate, high-fidelity emulation of work itself.

Since work is highly dependent on the payload contents, there is no good way to have accurate emulation anyways.

There is some planned work to have the tool use customer's per-recorded requests and replay them to solve this problem.

**Comment by guming3d** [comment]
Thanks for your feedback, do we have any feature plan to adding custom prompts from customer? Customer may want to use their prompt data for specific business scenario to do the test, from current code I can only modify the message generator function to do that.

**Comment by technicianted** [comment]
The work is planned but not committed. Contributions are welcome :)
